setup.cfg

```diff
@@ -0,0 +1,54 @@
+[metadata]
+name = wktplot
+version = attr: wktplot.__version__
```

Comment:
Are we keeping manual bump versioning for now?

Reply:
Keep manual bump versioning for now.